tap_s3_csv/s3.py

```diff
@@ -156,7 +157,7 @@ def get_input_files_for_table(config, table_spec, modified_since=None):
     matched_files_count = 0
     unmatched_files_count = 0
     max_files_before_log = 30000
-    for s3_object in list_files_in_bucket(bucket, prefix):
+    for s3_object in list_files_in_bucket(bucket, prefix, aws_endpoint_url=config['aws_endpoint_url']):
```

Comment:
`KeyError: 'aws_endpoint_url'` here as well when not in `config.json`

can we please use `.get` instead?

Reply:
updated.